app/controllers/test/CustomsDeclarationsStubController.scala

```diff
@@ -17,58 +17,20 @@
 package controllers.test
 
 import config.AppConfig
-import javax.inject.{Inject, Singleton}
-import models.Field._
-import models._
 import play.api.http.{ContentTypes, HeaderNames}
-import play.api.mvc.{Action, Codec, MessagesControllerComponents}
-import uk.gov.hmrc.http.HttpReads.Implicits._
+import play.api.mvc.{Codec, MessagesControllerComponents}
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
+import uk.gov.hmrc.http.HttpReads.Implicits._
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 
+import javax.inject.{Inject, Singleton}
 import scala.concurrent.ExecutionContext
-import scala.xml._
 
 @Singleton
 class CustomsDeclarationsStubController @Inject()(appConfig: AppConfig, httpClient: HttpClient, mcc: MessagesControllerComponents)(
   implicit ec: ExecutionContext
 ) extends FrontendController(mcc) {
 
-  def waiting(ref: String) = {
-    val customsDeclarationsConfig = appConfig.microservice.services.customsDeclarations
-    val customsDeclarationsBaseUrl =
-      s"${customsDeclarationsConfig.protocol.get}://${customsDeclarationsConfig.host}:${customsDeclarationsConfig.port.get}"
-
-    Waiting(
-      UploadRequest(
-        href = s"$customsDeclarationsBaseUrl/cds-file-upload-service/test-only/s3-bucket",
-        fields = Map(
-          Algorithm.toString -> "AWS4-HMAC-SHA256",
-          Signature.toString -> "xxxx",
-          Key.toString -> "xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-          ACL.toString -> "private",
-          Credentials.toString -> "ASIAxxxxxxxxx/20180202/eu-west-2/s3/aws4_request",
-          Policy.toString -> "xxxxxxxx==",
-          SuccessRedirect.toString -> s"$customsDeclarationsBaseUrl/cds-file-upload-service/upload/upscan-success/${ref}",
-          ErrorRedirect.toString -> s"$customsDeclarationsBaseUrl/cds-file-upload-service/upload/upscan-error/${ref}"
-        )
-      )
-    )
-  }
-
-  // for now, we will just return some random
-  def handleBatchFileUploadRequest: Action[NodeSeq] = Action(parse.xml) { implicit req =>
-    Thread.sleep(100)
-
-    val fileGroupSize = (scala.xml.XML.loadString(req.body.mkString) \ "FileGroupSize").text.toInt
-
-    val resp = FileUploadResponse((1 to fileGroupSize).map { i =>
-      FileUpload(i.toString, waiting(i.toString), id = s"$i")
-    }.toList)
-
-    Ok(XmlHelper.toXml(resp)).as(ContentTypes.XML)
-  }
-
```

Comment:
Why is it these functions are getting chopped out?

Reply:
These methods were used to service the now removed 'batch-file-upload' endpoint that is now located in the declaration stub service.